.buildkite/verify_c.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
this will run everything in serial. If you define this matrix in the pipline.yml then we can run builds in parallel.

Reply:
Fixed this myself.